server/app/views/applicant/ProgramIndexView.java

```diff
@@ -371,8 +371,12 @@ private PTag programCardApplicationStatus(
       Locale preferredLocale, StatusDefinitions.Status status) {
     return p().withClasses("border", "rounded-lg", "px-2", "py-1", "mb-4", "bg-blue-100")
         .with(
-            span(status.localizedStatusText().getOrDefault(preferredLocale))
-                .withClasses("text-xs", "font-medium"));
+            Icons.svg(Icons.INFO)
+                .withClasses("inline-block")
+                .withStyle("width: 18px; height: 18px;"),
```

Comment:
**suggestion (blocking):** recommend using tailwind classes rather than inline styles. If you need values that aren't provided out of the box you can use https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values

Reply:
This is what we're doing in other places like https://github.com/civiform/civiform/blob/7b89d2c75152a745fb9a63c075a3d7453df7b4c8/server/app/views/ViewUtils.java#L88-L89 and https://github.com/civiform/civiform/blob/7b89d2c75152a745fb9a63c075a3d7453df7b4c8/server/app/views/admin/programs/ProgramStatusesView.java#L267-L269.  If it seems okay, I can change those as well.

Reply:
bummer. well maybe we can figure out why at somepoint, feel free to ignore for now

Reply:
Actually, what we can do is add this to our tailwind.config.js
```
theme: {
  extend: {
    spacing: {
      4.5: '18px'
    }
  }
}
```
And then use `h-4.5` and `w-4.5`.  Works like a charm.

Reply:
nice!